### PR TITLE
Feature/genre pages

### LINKED ISSUE #6: Develop Filter Method to sort books by genre

### PR DESCRIPTION
**Pull request by orlov-n**
**[Project Ticket](Project ticket link goes here)**
https://github.com/users/mikeharrison57/projects/5#card-84506549

**Describe the changes made to the codebase:**

- Created BookGenrePage component
- Installed Router
- Minor HTML structure changes
- Added keys to iterations inside the components
- Fixed the CSS bug where two HTML elements overlapped with each other

**How was it tested?**

- Successfully rendered on the DOM

**Type of Change**

- [x] Bug Fix
- [x] New Feature
- [x] Refactor

**Issues**

- closes #6 #4 

**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

**Emoji of how you feel submitting this PR (optional):**

- Emoji:  🤙

### CODE REVIEW

**Review by megschuetz — APPROVED** [review]
Looks great!

File: src/BookGenrePage/BookGenrePage.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    )
+  })
+
+  return (
+    <>
+    {/* {{selectedGenreBooks} } */}
+    {/* {console.log('this is list name', listName)} */}
+    {/* {console.log('this is selectedGenre', selectedGenre)} */}
+    {/* <div>I am BookGenre Component</div>
+    <div>{listName}</div> */}
```
Comment (catlynbowles) [inline src/BookGenrePage/BookGenrePage.tsx:30]: Reminder to delete this commented out code ! But I love reusing the bookcards here, was super fun to learn about. 💅🏼

File: src/app/App.tsx [hunk 2]
```diff
@@ -21,15 +23,29 @@ class App extends React.Component<{}, IState> {
...
+          <main>
+            <TopBooks genres={this.state}/>
+          </main>
+       }
+       />
+
+       <Route exact path='/:list_name' render={({match}) => {
+        {console.log('this is match', match.params.list_name)}
+        return (<section className='book-genre-page'>
+          <BookGenrePage listName={match.params.list_name} genres={this.state} />
```
Comment (catlynbowles) [inline src/app/App.tsx:43]: YES Router ! 🐳

**Review by catlynbowles — APPROVED** [review]

**Review by mikeharrison57 — APPROVED** [review]
Amazing job with implementing Router Nick! It's great to see our page routing to the other genre pages. We will need to address the refresh bug, but we can cross that bridge tomorrow.

**Pull request merged**